src/server.rs

```diff
@@ -15,113 +14,150 @@ use crate::configuration::Configuration;
+                Ok(_bearer) => match self.http_verb {
+                    Method::Head => match req.param("key") {
+                        Some(key) => match &store.get(key.to_string()) {
+                            Some(_) => {
+                                res.set(StatusCode::Ok);
+                                res.send("")
+                            },
+                            None => {
+                                res.set(StatusCode::NotFound);
+                                res.send("")
+                            }
+                        },
+                        None => {
+                            res.set(StatusCode::BadRequest).set(MediaType::Json);
+                            res.send(serde_json::to_string_pretty(&ErrorMessage { code: 102, message: "Missing key parameter.", details: None }).unwrap())
+                        }
                     },
-                    None => {
-                        res.set(StatusCode::NotFound);
-                        res.send("")
-                    }
-                },
-                _ => {
-                    res.set(StatusCode::BadRequest).set(MediaType::Json);
-                    res.send(serde_json::to_string_pretty(&ErrorMessage { message: "Missing key parameter.".to_string(), details: None }).unwrap())
-                }
-            },
-            Method::Put => {
-                if body_size == 0 {
-                    res.set(StatusCode::BadRequest).set(MediaType::Json);
-                    return res.send(serde_json::to_string_pretty(&ErrorMessage { message: "Missing request body.".to_string(), details: None }).unwrap());
-                }
-                match req.param("key") {
-                    Some(key) => if buffer.len() < 7340032 {
-                        store.set(key.to_string(), buffer);
-                        res.set(StatusCode::Ok);
-                        res.send("")
-                    } else {
-                        res.set(StatusCode::BadRequest).set(MediaType::Json);
-                        res.send(serde_json::to_string_pretty(&ErrorMessage { message: "The maximum allowed value size is 7 Mb.".to_string(), details: None }).unwrap())
+                    Method::Put => {
+                        if body_size == 0 {
+                            res.set(StatusCode::BadRequest).set(MediaType::Json);
+                            return res.send(serde_json::to_string_pretty(&ErrorMessage { code: 101, message: "Missing request body.", details: None }).unwrap());
+                        }
+                        match req.param("key") {
+                            Some(key) => if buffer.len() < 7340032 {
+                                store.set(key.to_string(), buffer);
+                                res.set(StatusCode::Ok);
+                                res.send("")
```

Comment:
Following the documentation:

200 - OK
```json
{
  "message": "The specified key was successfully updated."
}
```

201 - Created
```json
{
  "message": "The specified key was successfully created."
}
```

See https://clintnetwork.gitbook.io/lucid/docs/api#store-data-create-and-update

Reply:
Yes, you right.